src/seedu/addressbook/parser/Parser.java

```diff
@@ -76,6 +62,9 @@ public Command parseCommand(String userInput) {
         case AddCommand.COMMAND_WORD:
             return prepareAdd(arguments);
 
+            case AddAndList.COMMAND_WORD:
```

Comment:
This indentation is not consistent with the other cases. It affects the readability of your code.